src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private static String[] colorsList = new String[]{"yellow", "purple", "orange",
+            "green", "green", "blue", "red", "pink"};
```

Comment:
let's make this a constant
```suggestion
    private static final String[] COLORS_LIST = new String[]{"yellow", "purple", "orange",
            "green", "green", "blue", "red", "pink"};
```

Reply:
done